addons/social_share/wizards/snshare_url_share_multi.py

```diff
@@ -0,0 +1,30 @@
+from markupsafe import Markup
+
+from odoo import fields, models
+
+
+class ShareWizard(models.TransientModel):
+    _name = 'snshare.url.share.multi'
```

Comment:
Side question: do we need another wizard ? Can't we make the base wizard being either single, either multi, and handle code gracefully in it (like we do in other composers) ?